mentoring/tests/integration/xml/mrq_with_html_choices.xml

```diff
@@ -1,24 +1,22 @@
 <vertical_demo>
-    <mentoring url_name="mcq_with_comments" display_name="MRQ Exercise 7" weight="1" enforce_dependency="false">
-        <title>MRQ With Resizable popups</title>
-        <mrq name="mrq_1_1_7" type="choices">
-            <question>What do you like in this MRQ?</question>
-            <choice value="elegance"><html><b>Its elegance</b></html></choice>
-            <choice value="beauty"><html><i>Its beauty</i></html></choice>
-            <choice value="gracefulness"><html><strong>Its gracefulness</strong></html></choice>
-            <choice value="bugs"><html><span style="font-color:red">Its bugs</span></html></choice>
+    <mentoring url_name="mcq_with_comments" display_name="MRQ With Resizable popups" weight="1" enforce_dependency="false">
+        <mrq name="mrq_1_1_7" question="What do you like in this MRQ?" required_choices="elegance,beauty,gracefulness">
+            <choice value="elegance"><b>Its elegance</b></choice>
```

Comment:
While we're here, can we use the likes of `<strong>` and `<em>`? This is a nit, but it would be nice.